.ci/on_conan_release.jenkinsfile

```diff
@@ -64,26 +64,44 @@ node('Linux') {
         }
 
         // Open PR in Github
+        int prNumber = null
         stage('Open PR in Github') {
-            def data = readJSON text: '{}'
+            Map data = [:]
             data.title = "[up conan] Update Conan version (${params.conan_version})" as String
             data.head = branchName
-            data.base = 'master' as String
+            data.base = 'master'
             data.body = '''\
                 New Conan version released.
 
-                Please, @conan-io/barbarians have a look to this PR
-
                 ---
 
                 <sup>This PR has been created automatically by CI</sup>
                 '''.stripIndent() as String
-            writeJSON json: data, file: 'data.json'
 
-            withCredentials([usernamePassword(credentialsId: 'conanci-gh-token', usernameVariable: 'GH_USER', passwordVariable: 'GH_PASS')]) {
-                String create_pr_url = 'https://api.github.com/repos/conan-io/conan-docker-tools/pulls'
-                sh script: "curl --fail --connect-timeout 20 --max-time 40 --retry 5 --retry-delay 10 --retry-max-time 40 -s -u $GH_USER:$GH_PASS --request POST ${create_pr_url} -d \"@data.json\""
-            }
+            String jsonContent = writeJSON(json: data, returnText: true)
+            Map response = script.httpRequest(
+                url: 'https://api.github.com/repos/conan-io/conan-docker-tools/pulls',
+                httpMode: 'POST',
+                authentication: 'conanci-gh-token',
+                timeout: 20,
+                requestBody: jsonContent,
+            )
+
+            Map<String, Object> responseData = readJSON(text: response.content)
+            prNumber = responseData.id as int
+        }
+
+        stage('Request reviewers') {
+            Map data = [:]
+            data.reviewers = ['uilianries', 'jgsogo']
```

Comment:
@SSE4  do you want to be a requested reviewer? You usually read CDT PRs as far as I see.

Reply:
I can try to do my best, if it helps moving things forward. as I can see there is not so much activity compare to CCI, so it shouldn't require that much attention.

Reply:
This affects only to this kind of PR created by the bot: https://github.com/conan-io/conan-docker-tools/pull/423, one every month.

Reply:
You can always add/remove you later, in any case.

@uilianries , please review (this one) and merge, we need to try it using CI